pypose/optim/optimizer.py

```diff
@@ -371,7 +378,8 @@ def __init__(self, model, solver=None, strategy=None, kernel=None, corrector=Non
         self.strategy = TrustRegion() if strategy is None else strategy
         defaults = {**{'min':min, 'max':max}, **self.strategy.defaults}
         super().__init__(model.parameters(), defaults=defaults)
-        self.jackwargs = {'vectorize': vectorize, 'flatten': True}
+        # self.jackwargs = {'vectorize': vectorize, 'flatten': True}
+        self.jackwargs = {'vectorize': vectorize, 'flatten': False}
```

Comment:
`flatten` is `False` doesn't support multiple Model parameters, please change back to `True`. will add an additional test for this.